alf/algorithms/entropy_target_algorithm.py

```diff
@@ -240,9 +242,9 @@ def adjust_alpha(self, entropy):
         target_entropy = self._target_entropy()
 
         if target_entropy > 0:
-            fast_stage_thresh = 0.5 * target_entropy
+            fast_stage_thresh = (0.5 * target_entropy).astype(np.float32)
```

Comment:
From my understanding, now we have to explicitly specify float32 whenever a floating number is involved?

Reply:
Whether it requires special attention to dtype mainly depends on the pytorch function under consideration.

This line is not needed here and has been removed. The actual conversion happens in L157-160, where we convert np.float64 to torch tensors. In pytorch1.4, it will return a tensor with type of torch.float32. But in 1.8, it will return torch.float64.  

In the concrete case of entropy target algorithm, when we apply torch.where(cond, a, b) laster, we will get an error since a and b are of different dtypes. 

In summary, in this case, because of the torch.where function, we have to pay attention to this conversion issue. For any other similar situations where such conversion is needed, we will get an error message from torch at least.